File: TextosJapones/src/main/java/org/jisho/textosJapones/util/converter/IntegerConverter.java
package org.jisho.textosJapones.util.converter;

import javafx.util.StringConverter;

public class IntegerConverter extends StringConverter<Integer> {

	@Override
	public String toString(Integer object) {
		if (object == null)
			return null;
		return String.valueOf(object);
	}

	@Override
	public Integer fromString(String string) {
		if (string == null || string.isEmpty())
			return null;
		return Integer.valueOf(string);
	}

}

File: ProcessaTexto/src/main/kotlin/br/com/fenix/processatexto/util/converter/IntegerConverter.kt
package br.com.fenix.processatexto.util.converter

import javafx.util.StringConverter


class IntegerConverter : StringConverter<Int?>() {

    override fun toString(integer: Int?): String? {
        return integer?.toString()
    }

    override fun fromString(string: String?): Int? {
        return if (string == null || string.isEmpty()) null else string.toInt()
    }
}